lib/DDG/Spice/Syntax.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Syntax;
+
+# ABSTRACT: Look up syntax for a programming language
+
+use DDG::Spice;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
+spice to => 'https://syntaxdb.com/api/v1/concepts/search?q=$1';
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers any => 'syntax', 'syntaxdb', 'syntaxcenter';
+
+# Handle statement
+handle remainder => sub {
+
+    # Query is in $_ or @_, depending on the handle you chose...if you
+    # need to do something with it before returning
+    return $_;
```

Comment:
For example all these would currently trigger this:

![syntax_for_dummies_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/15154777/351c3dca-1694-11e6-8b72-e1523c3a17e4.png)

![syntax_for_dummies_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/15154792/451e8c78-1694-11e6-8a2d-fe7d6a98bf93.png)